cmd/gitserver/build.sh

```diff
@@ -21,7 +23,15 @@ export CGO_ENABLED=0
 pkg="github.com/sourcegraph/sourcegraph/cmd/gitserver"
 go build -trimpath -ldflags "-X github.com/sourcegraph/sourcegraph/internal/version.version=$VERSION  -X github.com/sourcegraph/sourcegraph/internal/version.timestamp=$(date +%s)" -buildmode exe -tags dist -o "$OUTPUT/$(basename $pkg)" "$pkg"
 
-docker build -f cmd/gitserver/Dockerfile -t "$IMAGE" "$OUTPUT" \
+DOCKERFILE="cmd/gitserver/Dockerfile"
+for arg in "$@"; do
+  shift
+  case "$arg" in
+    '--ms-git')  DOCKERFILE="cmd/gitserver/Dockerfile_ms_git";;
```

Comment:
```suggestion
    '--microsoft-git')  DOCKERFILE="cmd/gitserver/Dockerfile_ms_git";;
```

Think of all the future code search users 😄